setup.py

```diff
@@ -145,7 +145,7 @@
     },
     package_data={
         "chia": ["pyinstaller.spec"],
-        "": ["*.clvm", "*.clvm.hex", "*.clib", "*.clinc", "*.clsp", "py.typed"],
+        "": ["*.clsp", "*.clsp.hex", "*.clvm", "*.clib", "*.clinc", "*.clsp", "py.typed"],
```

Comment:
`.clvm` still means something although it's something pretty rare now.  I just figured I should be conservative and keep it.  `.clinc` should go though and I got rid of the duplicate.

Reply:
In so much as we're looking forward, would there still be `.clvm.hex` files to include?  Including some intended things, but not all, makes it easier to overlook the others later.

Reply:
So we don't have any `.clvm` files now.  If we did the intent would be to remove other things that we are adjusting handling of here.  This half-forward-looking to `.clvm` files (of a new form) is sounding like noise at this point.  Either this intermediate state of `.clsp.hex` plus hashes is noise or the potential replacement for them is noise.

Reply:
Allowing `.clvm` files does not eliminate the need for `.hex` files, it's a separate issue.  We would not remove anything in their presence.

Reply:
> I think the point of doing a `.clvm` file would be actually to not have a `.hex` file because it would be so small it could just be loaded at runtime and the `.hex` would be more overhead than optimization.

I must be misunderstanding something.

> I think the point of doing a `.clvm` file would be actually to not have a `.hex`

> Allowing `.clvm` files does not eliminate the need for `.hex` files

Reply:
`.clvm` wouldn't need a hex file but `.clsp` still would